askama_derive/src/parser.rs

```diff
@@ -290,6 +291,38 @@ named!(arguments<Input, Vec<Expr>>, do_parse!(
     (args.unwrap_or_default())
 ));
 
+named!(macro_arguments<Input, &str>, alt!(
+    delimited!(char!('('), nested_parenthesis, char!(')')) |
+    map!(delimited!(char!('('), take_until!(");"), tag!(");")),
+        |s| str::from_utf8(&s).unwrap()
+    )
+));
+
+fn nested_parenthesis(i: Input) -> Result<(Input, &str), nom::Err<Input>> {
```

Comment:
I do think this should be extended to be robust against parentheses inside single or double unescaped quote characters.

Reply:
Then we will also have to deal with the escape cases' \ '' and "\"

Reply:
Ok, done with escape.